### DISCUSSION TITLE
How to add custom font in the NetBeans editor

### DISCUSSION THREAD

**Discussion by bhavinp311** [apache/netbeans#4655] [General]
Hi there,

I want to add a custom font in the Netbeans editor.

I want to change the font to Menlo but it is not there in the list (Tools->Options->Fonts & Colors).

How can I achieve that?

Does anyone know please let me know?

Thanks!

**Comment by moacirrf** [comment]
1. Install this font on your operational system
2. Restart netbeans, and font will be available:
    Tools > Options > Fonts and Colors
    ![image](https://user-images.githubusercontent.com/950706/191488669-32dc8a00-8bba-48d9-ab68-6bbf63ef6b72.png)
